CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # CHANGELOG
 
+<<<<<<< HEAD
```

Comment:
Looks like a rebase gone bad?

Once a review has started, can we please not rebase? It makes point to point reviews of code changed impossible as the git references get blown away.

Reply:
Wait, I did push a fix

Reply:
Ugh, I've force-pushed so many times I'm ashamed of myself rn

Reply:
That was my fault, was fixing a conflict in the CHANGELOG.

Reply:
Noice